README.md

```diff
@@ -1,51 +1,87 @@
 # hmpo-i18n
-i18n for node
 
-A ground up rebuild of the [i18next](http://www.npmjs.com/package/i18next) project for a node environment.
+A lightweight internationalization (i18n) middleware for Node.js and Express applications, built to support His Majesty’s Passport Office (HMPO) services. Based on similar concepts to [i18next](https://www.i18next.com/). 
 
-The [i18next](http://www.npmjs.com/package/i18next) project has a number of issues that result from it being a port of an originally client-side project. In particular that it maintains a single global datastore, and so if two modules resolve to the same instance then they will interfere with each other.
+---
 
-The aim of this project is to create a module-safe, lightweight translation library for a node environment, based on similar concepts to i18next.
+## Features 
 
-## Install
+### Translation Management
+- **JSON-based storage**: Simple key/value pairs in structured JSON files
+- **Nested keys**: Organize translations hierarchically (e.g., `"buttons.submit": "Submit"`)
+
+### Locale Handling
+- **Automatic detection**: Supports multiple lookup sources:
+    - 🍪 Cookies
+    - 🔗 Query parameters
+    - 🌐 HTTP headers (`Accept-Language`)
+- **Express middleware**: Zero-config integration for route handling
+- **Multi-source loading**: Load translations from project directories and `node_modules`
+
+## Installation 
 
 ```
 npm install hmpo-i18n
 ```
 
-## Usage
+## Basic Setup
 
-First create some resource files. These should be json or yaml files and the location of the file within your project will define the language it corresponds to.
+#### 📁 Folder Structure
```

Comment:
Can with add some text like was here to show you can use yaml files or json. And similarly include the details about the resource path